src/main/java/frc/robot/util/MoPrefs.java

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.util;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableEvent;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+import edu.wpi.first.networktables.StringPublisher;
+import java.util.EnumSet;
+import java.util.function.BiFunction;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+/** Robot preferences, accessible through Shuffleboard */
+public class MoPrefs {
+    public static final Pref<Double> maxDriveSpeed = doublePref("Drive Max Speed m/s", 6000.0);
+    public static final Pref<Double> maxTurnSpeed = doublePref("Turn Max Speed m/s", 6000.0);
```

Comment:
It has always bothered me that these preferences have slashes (`/`) in their name, since it causes the shuffleboard to treat them as a tree of preferences. But I never got around to renaming them, since I didn't want to lose the saved values.

Let's take the opportunity to rename these variables now.

Reply:
Maybe we could add some logic to the constructor of Pref to reject/fix names that contain slashes?